web-client/src/components/DashboardLayout/DashboardLayout.tsx

```diff
@@ -22,7 +23,7 @@ const DashboardLayout: React.FC<DashboardLayoutProps> = ({
   const [notificationVisible, setNotificationVisible] = useState(false);
 
   return (
-    <Layout>
+    <StyledLayout>
```

Comment:
Use a descriptive name.  Styled for what purpose?  What is this component doing?  Treat it like any other React component and give it a real name.

Reply:
I considered this, but `Layout` is Antd component so, I approached here to indicate this is the styled version of Antd `Layout` component. The reason is, we'll need to write some media queries for the desktop layout.

Reply:
It is very hard to work with and refactor code if you use generic names like "Layout"
This is a ReactComponent and should be named like any other component
If you wan to indicate inheritance relationship with antd.Layout, call it "MapDisplayLayout"

Reply:
This is in the `DashboardLayout` component, not in the request module, or isn't only related to this screen, so `MapDisplayLayout` is not an option here. IMO, this is the only place we can use some generic names as it's responsible for the entire layout system.